components/style/themes/default.less

```diff
@@ -12,6 +12,8 @@
 @error-color            : @red-6;
 @highlight-color        : @red-6;
 @warning-color          : @gold-6;
+@default-color          : #fff;
+@alternate-color        : #000;
```

Comment:
I don't really understand what `alternate` stands for which is sometimes white and sometimes black.

Reply:
Maybe `default-light-color` and `default-dark-color` would be better, or even `@black` and `@white`.

I added these two in order to be able to replace the harcoded shades of grey by `shade(@black, x%)`, which has the benefit of working with any underlying background (as opposed to `#444` for instance, which assumes that the background is white and thus does not work if `@component-background` is assigned another value).

Reply:
`@black` & `@white` is nice and simple enough.